frontend/src/components/layouts/CardLayout.js

```diff
@@ -0,0 +1,79 @@
+import React from 'react'
+import PluginCard from '../PluginCards/pluginCard'
+import { Container, Row, Col } from 'reactstrap';
+
+
+class CardLayout extends React.Component {
+
+    state = {
+        people: [
```

Comment:
```suggestion
        plugins: [
```

Reply:
Oh that was dumb on my end, I was just watching a tutorial :)